app/controllers/application_controller.rb

```diff
@@ -2,7 +2,5 @@
 
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
-  if Rails.env="test"
-    skip_before_action :verify_authenticity_token
-  end
+  skip_before_action :verify_authenticity_token if Rails.env.test?
```

Comment:
here its better to use
```
  skip_before_action :verify_authenticity_token if (Rails.env = test)
```
its just a bug in rails, test? method doesnt work properly